.travis.yml

```diff
@@ -82,22 +81,25 @@ script:
         make install )
 
 after_success:
-    # Push coverage info on coveralls.io.
+    # Push coverage info on codecov.io.
     # Ignore generated files, samples and tests
     - if [ "${coverage}" = "ON" ]; then
-          coveralls --exclude "build_debug/bindings/python"
-                    --exclude "build_debug/CMakeFiles"
-                    --exclude "build"
-                    --exclude "install"
-                    --exclude "skeleton-subsystem"
-                    --exclude "tools/clientSimulator"
-                    --exclude "test/test-subsystem"
-                    --exclude "test/functional-tests"
-                    --exclude "bindings/c/Test.cpp"
-                    --exclude "test/tokenizer"
-                    --exclude "utility/test"
-                    --gcov /usr/bin/gcov-4.8
-                    --gcov-options '\--long-file-names --preserve-paths';
+          bash <(curl -s https://codecov.io/bash)
```

Comment:
- Why use a process substitution instead of a regular pipe ?
 - I would rather do the curl on a separate line in order to detect and explicit download failure
 - Why using curl where the rest of the script uses wget (and long options)